app/view_models/workarea/storefront/search_autocomplete_view_model.decorator

```diff
@@ -0,0 +1,15 @@
+module Workarea
+  if Plugin.installed?(:search_autocomplete)
+    decorate Storefront::SearchAutocompleteViewModel, with: :content_search do
+      def content_results
+        return [] if searches.blank?
+
+        results = Storefront::ContentSearchViewModel.wrap(response, options)
+
+        results.content.take(
+          Workarea.config.storefront_search_autocomplete_max_products
```

Comment:
Might as well make a new config for this so it can be controlled separately - content display isn't really tied to products.